templates/CRM/Contact/Import/Form/MapField.tpl

```diff
@@ -28,7 +28,7 @@ if ( document.getElementsByName("saveMapping")[0].checked ) {
     document.getElementsByName("saveMapping")[0].checked = false;
 }
 {/literal}
-{if $isCheked}
+{if isset($isCheked) && $isCheked}
```

Comment:
Believe it or not, the variable is spilled consistently.

Reply:
that won't work :-( it will cause a fatal with smarty is secure mode :-(

I broke my brain on that particular notice so I was really excited when I saw you had tried but this isn't the answer (I concluded re-writing the js probably was)

Reply:
Yeah, the life on that flag is weird.

Is there a configuration flag that puts into the non-working mode?

Reply:
yeah  - you can also use `{if $isCheked|smarty:nodefautls}`